app/models/edm/web_resource.rb

```diff
@@ -140,6 +147,13 @@ def europeana_supported_media_mime_type
       end
     end
 
+    def media_size
+      limit = MAX_MEDIA_SIZE.to_f
+      if media.file.size.to_f > limit
```

Comment:
Why `#to_f`? File sizes and limits are integers.

Reply:
on the example from the carrierwave wiki (https://github.com/carrierwaveuploader/carrierwave/wiki/How-to%3A-Validate-image-file-size) where I based this off they're dividing the file size, so it wouldn't work with integers I guess. This is just left over from that, doesn't seem to be necessary for us.